studiengaenge/bachelor_informatik.tex

```diff
@@ -65,29 +65,29 @@ \subsection{Prüfungen: wie und wieso?}
 
 Häufig habt ihr pro Prüfung zwei Versuche, es ist also nicht so schlimm, wenn beim ersten Versuch nicht bestanden habt. Einige Dozierende weichen aber von der Standardregelung ab, teilen euch dies allerdings direkt am Anfang des Semesters mit. Informiert euch also am \emph{Anfang des Semesters} immer genau über die Bedingungen! Solltet ihr eine Prüfung auch beim Zweitversuch nicht bestehen, so besteht in bis zu vier Fällen die Möglichkeit, die Prüfung \emph{auf Antrag beim Prüfungsausschuss} ein weiteres Mal zu wiederholen (gilt nicht für Orientierungsprüfung und Bachelorarbeit). Besteht ihr auch diesen Versuch nicht, verliert ihr euren Prüfungsanspruch endgültig und werdet exmatrikuliert. Dies und weitere studienrelevante Vorschriften sind in der Prüfungsordnung geregelt, deshlab lohnt es sich definitiv, die Prüfungsordnung einmal komplett zu lesen. Außerdem kann sich diese leider immer mal wieder ändern, darauf sollte man achten.
```

Comment:
Dieser Satz ist sehr missverständlich. Da Prüfung in der neuen PO == Klausur gilt, würde ich Folgendes vorschlagen: Häufig habt ihr pro Modul zwei Klausren, ... [...]. In manchen Grundlagenmodulen habt ihr sogar vier Klausuren, informiert euch also am ....